### DISCUSSION TITLE
XR control

### DISCUSSION THREAD

**Discussion by swapneshdev** [pmndrs/xr#330] [Q&A] [answered]
Can we add orbitalcontrol from react-three/drei to the Model when is AR view/environment ?  (to enable move ,rotate and other functionalities to the model)

Could you guide me on how to implement this?

Thank you!

**Comment by bbohlender** [comment, marked as answer]
Orbit Controls are not controller individual models, but the camera, which is therefore unsuitable for AR/VR experiences. I guess you are referenecing to DragControls for transforming individual models.

As seen in the roadmap, this is not yet implemented. Building this yourself is rather tricky, so I recommend waiting for the official implementation, but here's a tutorial on how to get started with dragging a cube: https://docs.pmnd.rs/xr/tutorials/interactions#pointer-capture
